ship_traffic/ship_traffic.ipynb

```diff
@@ -509,7 +514,7 @@
     "sopts         = dict(start=0, end=1, sizing_mode='stretch_width')\n",
     "map_opacity   = pn.widgets.FloatSlider(value=0.7, name=\"Map opacity\",   **sopts)\n",
     "data_opacity  = pn.widgets.FloatSlider(value=1.0, name=\"Data opacity\",  **sopts)\n",
-    "label_opacity = pn.widgets.FloatSlider(value=0.9, name=\"Label opacity\", **sopts)\n",
+    "label_opacity = pn.widgets.FloatSlider(value=0.3, name=\"Label opacity\", **sopts)\n",
```

Comment:
The labels are a bit hard to read on this map. Is it because they're too transparent?
<img width="989" alt="image" src="https://github.com/user-attachments/assets/708e3b1d-a429-40c2-aef5-99ee14019feb">

Reply:
Agreed. Maybe the CartoDB.VoyagerOnlyLabels would show up better? I haven't tried it myself.

Also, what's up with the data rendering resolution? :-)

Reply:
> Also, what's up with the data rendering resolution? :-)

It's a screenshot from the website after zooming in :)

Reply:
> The labels are a bit hard to read on this map. Is it because they're too transparent?

Yes, it's the map opacity. I've put the default back to 0.7 now

#### PositronOnly
![ship_t_pos7](https://github.com/user-attachments/assets/de766ed9-b8ee-46d7-95e1-0e580d610854)

#### VoyagerOnly
![ship_t_VO7](https://github.com/user-attachments/assets/2c95d4be-5990-4f0b-891c-8e37ec54d7d6)

Reply:
I can't spot a difference between those two; are you sure they are different?

Reply:
I'm sure. I too, can't see any visual difference.